yax/core/src/main/scala/doobie/hi/resultset.scala

```diff
@@ -206,7 +206,7 @@ object resultset {
       val b = Vector.newBuilder[A]
       while (n > 0 && rs.next) {
```

Comment:
If you're doing that there's not much reason to be using a stream, but yeah we should review the doc. Opened #322 for followup.